src/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://unpkg.com/boxicons@2.1.2/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <title>LeaderBoards</title>
+  </head>
+  <body class="flex">
+    <main class="container flex">
+      <h1>
+        <i class="bx bxs-star"></i><i class="bx bxs-star"></i
+        ><i class="bx bxs-star"></i> LeaderBoard
+      </h1>
```

Comment:
- [ ] In the wireframe layout, the "Leaderboard" heading is all the way to the left but yours is centered. Kindly adjus it to match the wireframe layout
### Your Layout
![image](https://user-images.githubusercontent.com/46839548/187046823-05159187-83df-44f6-8dde-be2f584708d1.png)
### Wirefame layout
![image](https://user-images.githubusercontent.com/46839548/187046845-c1f99f41-0b3b-4213-ba42-3f3c4afebedc.png)